dialog-db/starting-dialog.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Dialog - Personal Substrate for Cooperative Computing
+date: 2025-08-12
+---
+
+# Dialog - Personal Substrate for Cooperative Computing
+
+You're planning dinner for friends this weekend. Your calendar knows who's coming. Your contacts know Sarah's gluten-free, Tom's vegetarian. As you open a meal planner, it suggests recipes everyone can enjoy, portions adjusted for your group. Your shopping list populates with needed ingredients.
```

Comment:
I think this would be stronger if you paint the idyllic vision then turn it to the user: "why don't we already have this?" Obvious status-quo answer: "some company has to decide it's worth it to them and their investors".

Naturally segues into the barriers to cooperation created by exisiting models.